### PR TITLE
update readme to include the `key` and add `--config` arg to first example

### PR DESCRIPTION
**Pull request by sallyom**
/cc @ondrejbudai @achilleas-k

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,6 +254,29 @@ Example:
 }
 ```
 
+### Accessing the system
+
+With a virtual machine launched with the above [virt-install](#virt-install) example, access the system with
+
+```
+ssh -i /path/to/private/ssh-key foo@ip-address
```
Comment (achilleas-k) [inline README.md:262]: We have two example configs, one with username `foo` for the general build config example and one with `alice` for the user specific example.  I doubt anyone will be confused by this but maybe we should have just one username.

**Review by achilleas-k — APPROVED** [review]
Thanks so much!  Small comment below about the usernames in the examples but I think we can fix it later.

File: README.md (hunk 1 above)
Comment (sallyom) [reply README.md]: let's go with alice! i like that better :) i pushed the update...

**Review by teg — DISMISSED** [review]

**Review by achilleas-k — APPROVED** [review]
Thanks so much!!

**Comment by achilleas-k** [comment]
Rebased and resolved the conflicts with `main`.

**Review by achilleas-k — APPROVED** [review]

**Pull request merged**